src/utils/lm/install.js

```diff
@@ -181,6 +192,13 @@ const getCurrentCredentials = async () => {
   }
 }
 
+// Git expects the config path to always use / even on Windows
+const getGitConfigContent = (gitConfigPath) => `
+# This next lines include Netlify's Git Credential Helper configuration in your Git configuration.
+[include]
+  path = ${path.posix.normalize(gitConfigPath)}
```

Comment:
This might not work as expected.
`path.posix.normalize()` actually returns backslashes on Windows (which is confusing).

Is the intent to only use forward slashes? If so, I have personally used this unmaintained package: https://github.com/jonschlinkert/unixify (we use it in `zip-it-and-ship-it`). If you find a better package, I'd be curious to learn about it too :)